server/protocols/webadmin/webadmin-mailbox/src/test/java/org/apache/james/webadmin/routes/MailboxesRoutesTest.java

```diff
@@ -112,6 +113,7 @@ void beforeEach() {
     @AfterEach
     void tearDown() {
         webAdminServer.destroy();
+        taskManager.stop();
```

Comment:
:+1:

Reply:
I believe that with current scheduler we can not await 80 task at the same time.

I also suspect a thread leak upon await being the root cause of the issue.